Calendar/CalLogHelpers/CalLogCSVFunctions.ps1

```diff
@@ -193,37 +188,26 @@ function BuildCSV {
             'MeetingRequestType'             = $CalLog.MeetingRequestType.ToString()
             'StartTime'                      = ConvertDateTime($CalLog.StartTime)
             'EndTime'                        = ConvertDateTime($CalLog.EndTime)
+            'OriginalStartDate'              = ConvertDateTime($CalLog.OriginalStartDate)
             'TimeZone'                       = $CalLog.TimeZone
             'Location'                       = $CalLog.Location
-            'ItemType'                       = $ItemType
             'CalendarItemType'               = $CalLog.CalendarItemType.ToString()
             'IsException'                    = $CalLog.IsException
             'RecurrencePattern'              = $CalLog.RecurrencePattern
             'AppointmentAuxiliaryFlags'      = $CalLog.AppointmentAuxiliaryFlags.ToString()
             'DisplayAttendeesAll'            = $CalLog.DisplayAttendeesAll
             'AttendeeCount'                  = ($CalLog.DisplayAttendeesAll -split ';').Count
             'AppointmentState'               = $CalLog.AppointmentState.ToString()
-            'ResponseType'                   = $ResponseType
-            'SentRepresentingEmailAddress'   = $CalLog.SentRepresentingEmailAddress
-            'SentRepresentingSMTPAddress'    = GetSMTPAddress($CalLog.SentRepresentingEmailAddress)
-            'SentRepresentingDisplayName'    = $CalLog.SentRepresentingDisplayName
-            'ResponsibleUserSMTPAddress'     = GetSMTPAddress($CalLog.ResponsibleUserName)
-            'ResponsibleUserName'            = $CalLog.ResponsibleUserName
-            'SenderEmailAddress'             = $CalLog.SenderEmailAddress
-            'SenderSMTPAddress'              = GetSMTPAddress($CalLog.SenderEmailAddress)
```

Comment:
We don't want to know about the sender who requested the change?

Reply:
These are all the RAW forms of information.  The Processed fields are output earlier, Organizer, From, Sender, etc. These were here in case the other did not work, but they are still preserved in the RAW output, so they are just duplicate.